packages/core/src/components/buttons/buttons.module.scss

```diff
@@ -104,22 +126,22 @@
 
 	&:hover {
 		background: $colors-danger-3 !important;
-		color: white;
+		border-color: $colors-danger-3 !important;
+		color: $white;
 	}
 
 	&:focus {
-		background: $colors-warning-1!important;
 		color: $colors-neutral-8;
-		box-shadow: 0px 2px 0px $colors-neutral-8;
 	}
-}
-
-.intent-warning:disabled {
-	cursor: not-allowed !important;
-	background: $colors-danger-1;
 
-	&:hover {
-		background: $colors-neutral-6 !important;
+	&:disabled {
+		cursor: not-allowed !important;
+		background: $colors-danger-1;
+	
+		&:hover {
+			background: $colors-neutral-6 !important;
```

Comment:
probably this one should stay as `$colors-danger-1` when `disabled->hover`

Reply:
Fixed in PR #281